app/view/renderer.rb

```diff
@@ -5,6 +5,8 @@
 
 class Renderer
 
+  YEARS = [1, 2, 3, 4].freeze
+
   def initialize(topics, base_path = '')
```

Comment:
base_path as parameter? better const ?

Reply:
it can't be const the path changes during testing

Reply:
well well ... then add a getter method which points to the const. then override it in your specs. try to avoid adjusting code only because you wanna test it. mocking is pretty simple in ruby or rspec ;)